snuba/api.py

```diff
@@ -304,9 +304,7 @@ def parse_and_run_query(dataset, request: Request, timer) -> QueryResult:
         )
 
     source = dataset.get_dataset_schemas().get_read_schema().get_data_source()
-    # TODO: consider moving the performance logic and the pre_where generation into
-    # ClickhouseQuery since they are Clickhouse specific
```

Comment:
Does the removal of this comment, as well as the formatted query now being the only state stored on the `ClickhouseQuery`, mean that we're no longer going to do this?

Reply:
No, it means I deleted the wrong comment. I should have moved it instead. The content is still valid.